plugin.cpp

```diff
@@ -38,6 +38,34 @@ static const char * def_cfg = QUOTE({
 		"order" : "2",
 		"displayName" : "To "
 		},
+	"email_cc" : {
+		"description" : "The address to send the alert to Cc",
+		"type" : "string",
+		"default" : "alert.subscriber@dianomic.com",
+		"order" : "1",
+		"displayName" : "Cc address"
+		},
+	"email_cc_name" : {
+		"description" : "The name to send the alert to Cc",
+		"type" : "string",
+		"default" : "Notification alert subscriber",
+		"order" : "2",
+		"displayName" : "Cc "
+		},
+	"email_bcc" : {
+		"description" : "The address to send the alert to Bcc",
+		"type" : "string",
+		"default" : "alert.subscriber@dianomic.com",
+		"order" : "1",
+		"displayName" : "Bcc address"
+		},
+	"email_bcc_name" : {
+		"description" : "The name to send the alert to Bcc",
+		"type" : "string",
+		"default" : "Notification alert subscriber",
+		"order" : "2",
+		"displayName" : "Bcc "
+		},	
 	"subject" : {
```

Comment:
We probably should have `$NOTIFICATION_INSTANCE_NAME$` and/or `$REASON$` macro in subject